### ISSUE TITLE
TypeError: Cannot read property 'callee' of null

### ISSUE THREAD

**Issue by yankri** [yannickcr/eslint-plugin-react#2745]
Getting this error TypeError: Cannot read property 'callee' of null when calling `forwardRef(this.ref)`

```
componentDidMount() {
    const { forwardRef } = this.props;

    this.ref.current.focus();

    if (typeof forwardRef === 'function') {
      forwardRef(this.ref);
    }
}
```

Stack trace: 
TypeError: Cannot read property 'callee' of null
Occurred while linting file
    at Object.isDestructuredFromPragmaImport (/node_modules/eslint-plugin-react/lib/util/Components.js:348:33)
    at Object.isPragmaComponentWrapper (node_modules/eslint-plugin-react/lib/util/Components.js:604:69)

We solved this by eliminating the props destructuring and using `this.props.fowardedRef(this.ref)` but this seems to be an issue with the package.

**Comment by ljharb** [comment]
@yankri what version of eslint and eslint-plugin-react do you have installed?

**Comment by proxi** [comment]
I am seeing similar crash with `eslint 7.6.0` `eslint-plugin-react 7.20.5`:

```TypeError: Cannot read property 'callee' of null
    at Object.isDestructuredFromPragmaImport (/node_modules/eslint-plugin-react/lib/util/Components.js:348:33)
    at Object.isCreateElement (/node_modules/eslint-plugin-react/lib/util/Components.js:393:16)
    at Object.isReturningJSX (/node_modules/eslint-plugin-react/lib/util/Components.js:467:47)
```

**Comment by yankri** [comment]
@ljharb `eslint 6.8.0` and `eslint-plugin-react 7.20.5`

**Comment by ljharb** [comment]
This is solved by 8466625dee3ac366a575b27e3c380198733c0d42, which is currently unreleased. Duplicate of #2733.
